mmv1/third_party/terraform/tests/resource_dns_record_set_test.go.erb

```diff
@@ -142,9 +99,40 @@ func TestAccDNSRecordSet_changeType(t *testing.T) {
 			{
 				Config: testAccDnsRecordSet_bigChange(zoneName, 600),
 			},
+			{
+				ResourceName:  "google_dns_record_set.foobar",
+				ImportStateId: fmt.Sprintf("%s/%s/test-record.%s.hashicorptest.com./CNAME", getTestProjectFromEnv(), zoneName, zoneName),
+				ImportState:   true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccDNSRecordSet_rrdatasUpdate(t *testing.T) {
```

Comment:
Not sure I follow how this is distinct from `TestAccDNSRecordSet_Update`?

Reply:
Missed that `rrdatas` was being updated, removed the duplicate test.